sdk/script/management-sdk.ts

```diff
@@ -58,18 +44,24 @@ function urlEncode(strings: string[], ...values: string[]): string {
     return result;
 }
 
-export class AccountManager {
+class AccountManager {
+    public static PERMISSIONS = {
```

Comment:
Do we want this name to be uppercased? I had thought we were naming enums using sentence-casing, with their members being uppercased (e.g. InstallMode.IMMEDIATE).